docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,134 @@
+    ```
+
+### Setup Kyma Telemetry
+
+Enable ingestion of the signals from your workloads, using the available features of the Kyma Telemetry module:
+
+1. Enable a LogPipeline which is shipping container logs of all workload directly to the AWS X-Ray service, leveraging the same secret as the AWS Distro is using, bypassing the AWS Distro. As logs are not yet supported by the AWS Distro and the Kyma feature is not based on OTLP yet, the integration here is not consistent with the rest for now. For that replace `{NAMESPACE}` and enable the LogPipeline by running:
+    ```bash
+    kubectl apply -f ./resources/logpipeline.yaml
+    ```
+1. Enable a TracePipeline in the cluster so that all components have a well-defined OTLP based push URL in the cluster to send trace data to. For that replace `{NAMESPACE}` and apply a TracePipeline by calling 
+    ```bash
+    kubectl apply -f ./resources/tracepipeline.yaml
+    ```
+1. Enable a MetricPipeline in the cluster so that all components have a well-defined OTLP based push URL in the cluster to send metric data to. Also it will activate annotation based metric scraping for workloads. For that replace `{NAMESPACE}` and apply a MetricPipeline by calling
+    ```bash
+    kubectl apply -f ./resources/metricpipeline.yaml
+    ```
+
+## Verifying the results by deploying sample apps
+
+In order to verify the results of CloudWatch and X-Ray we will deploy sample applications for each service accordingly.
+
+### Verifying CloudWatch traces, logs, and metrics arrival 
+
+In order to deploy sample app which generates traces that we took from [aws-otel tutorial](https://docs.aws.amazon.com/eks/latest/userguide/sample-app.html):
+1. Deploy traffic generator app
+    ```bash
+    kubectl apply -n ${KYMA_NS} -f ./sample-app/traffic-generator.yaml
+    ```
+1. Deploy an app using 
+    ```bash
+    kubectl apply -n ${KYMA_NS} -f ./sample-app/deployment.yaml
+    ```
+1. Port-forward an application in order to be able to access it by calling 
+    ```bash
+    kubectl -n ${KYMA_NS} port-forward svc/sample-app 4567
+    ```
+1. Make some requests to the application like `localhost:4567` or `localhost:4567/outgoing-http-call`
+1. Go to the `AWS X-Ray` tab, and check out the `Traces` section
+1. To verify the logs, you can go to `AWS CloudWatch`, then open the `Log groups` and select your cluster. Now, you can open `aws-integration.sample-app-*` and check out the logs of your application.
+1. To verify metrics, you can go to the `All metrics`, and open the `aws-integration/otel-collector`
+
+### Creating the dashboard to observe incoming metrics and logs
+
+In order to create a dashboard, you should:
+1. Go to the `Dashboards` section
+1. Click `Create dashboard` and enter the name
+1. Select the widget type and click `Next`
+1. Select what you want to observe, either metrics or logs, and click `Next`
+1. Decide on which metrics and logs to include and click `Create widget`
```

Comment:
In the previous step it was either metrics or logs, so now I'm confused by the "and". Can users really select _both_ here?

```suggestion
1. Decide which metrics or logs to include and click **Create widget**.
```